SU2_CFD/src/solvers/CEulerSolver.cpp

```diff
@@ -2308,7 +2308,10 @@ void CEulerSolver::Upwind_Residual(CGeometry *geometry, CSolver **solver_contain
     /*--- Roe Turkel preconditioning ---*/
 
```

Comment:
```suggestion
    su2double limiter_i[MAXNVAR] = {0.0}, limiter_j[MAXNVAR] = {0.0}, ZeroVec[MAXNVAR] = {0.0};

    /*--- Roe Turkel preconditioning ---*/

```

Reply:
github does not let me get this change in but it should fix the dangling pointer issues.

Anyway... I'll leave this PR for the stale bot now